beta-architecture/diagrams/tdr-beta-high-level-architecture.svg

```diff
@@ -1 +1 @@
 transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aW"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#aG"/><use transform="matrix(0.05,0,0,0.05,21.950000000000003,0)" xlink:href="#az"/><use transform="matrix(0.05,0,0,0.05,31.950000000000003,0)" xlink:href="#aA"/></g><path d="M24-231v-30h32v30H24zM24 0v-190h32V0H24" id="aX"/><g id="l"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aU"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#aT"/><use transform="matrix(0.05,0,0,0.05,22,0)" xlink:href="#aE"/><use transform="matrix(0.05,0,0,0.05,36.95,0)" xlink:href="#aX"/><use transform="matrix(0.05,0,0,0.05,40.900000000000006,0)" xlink:href="#aL"/></g><g id="m"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aJ"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#aM"/></g><path d="M30-248c87 1 191-15 191 75 0 78-77 80-158 76V0H30v-248zm33 125c57 0 124 11 124-50 0-59-68-47-124-48v98" id="aY"/><path d="M177-190C167-65 218 103 67 71c-23-6-38-20-44-43l32-5c15 47 100 32 89-28v-30C133-14 115 1 83 1 29 1 15-40 15-95c0-56 16-97 71-98 29-1 48 16 59 35 1-10 0-23 2-32h30zM94-22c36 0 50-32 50-73 0-42-14-75-50-75-39 0-46 34-46 75s6 73 46 73" id="aZ"/><g id="n"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aY"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#aD"/><use transform="matrix(0.05,0,0,0.05,22,0)" xlink:href="#aG"/><use transform="matrix(0.05,0,0,0.05,31,0)" xlink:href="#aH"/><use transform="matrix(0.05,0,0,0.05,36,0)" xlink:href="#aZ"/><use transform="matrix(0.05,0,0,0.05,46,0)" xlink:href="#aA"/><use transform="matrix(0.05,0,0,0.05,51.95,0)" xlink:href="#az"/><use transform="matrix(0.05,0,0,0.05,61.95,0)" xlink:href="#aG"/></g><path d="M115-194c55 1 70 41 70 98S169 2 115 4C84 4 66-9 55-30l1 105H24l-1-265h31l2 30c10-21 28-34 59-34zm-8 174c40 0 45-34 45-75s-6-73-45-74c-42 0-51 32-51 76 0 43 10 73 51 73" id="ba"/><g id="o"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aW"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,22.950000000000003,0)" xlink:href="#aN"/><use transform="matrix(0.05,0,0,0.05,26.900000000000002,0)" xlink:href="#aD"/><use transform="matrix(0.05,0,0,0.05,36.900000000000006,0)" xlink:href="#aB"/><use transform="matrix(0.05,0,0,0.05,46.900000000000006,0)" xlink:href="#aT"/></g><path d="M0 4l72-265h28L28 4H0" id="bb"/><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#bb" id="p"/><g id="q"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aJ"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#aX"/><use transform="matrix(0.05,0,0,0.05,16.900000000000002,0)" xlink:href="#aA"/><use transform="matrix(0.05,0,0,0.05,22.85,0)" xlink:href="#aH"/><use transform="matrix(0.05,0,0,0.05,27.85,0)" xlink:href="#aK"/></g><g id="r"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aM"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#aS"/><use tra
```

Comment:
Typo in one of the "copy to clean" arrows: "Retirieve files".

Reply:
Resolved

Reply:
Looks like the typo's back? Or maybe there was more than one? It's on the arrow from the copy-to-clean Lambda, and the upload bucket.

Also, I've just noticed that that arrow points in the wrong direction.

Reply:
resolved